src/components/ChatLog.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './ChatEntry.css';
+import PropTypes from 'prop-types';
+import { DateTime } from 'luxon';
+import ChatEntry from './ChatEntry';
+
+
+
+const ChatLog = (props) => {
+    const {entries} = props;
```

Comment:
To save a line here, we can also destructure inside of the component call, like this: 
```
const ChatLog = ({entries}) => { ...
```